widgetssdk/src/main/res/layout/visitor_video_floating_view.xml

```diff
@@ -12,7 +12,35 @@
     tools:cardBackgroundColor="@android:color/black">
 
     <FrameLayout
-        android:id="@+id/visitor_video_container"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent">
+
+        <FrameLayout
+            android:id="@+id/visitor_video_container"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent" />
+
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
could you use `ConstraintLayout` instead top-level FrameLayout and have less nesting?

Reply:
I can remove this nesting level leave only container and textview if this will be ok (still in framelayout)